integral_view/urls.py

```diff
@@ -38,6 +38,8 @@
 
 from django.contrib.auth.decorators import login_required
 
+from integral_view.views.system_date_time import view_date_time, update_date_time
```

Comment:
Now that we have more views related to the system page, lets move these into a view file called system.py rather than using a view file specific to time and date.